lib/codecs/UrlCodec.js

```diff
@@ -2,6 +2,7 @@
  * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
  */
 import * as b58 from 'base58-universal';
+import * as uuid from 'uuid';
```

Comment:
This library pulls in lots of things like ... a sha1 and md5 implementation that we definitely don't need. It says it supports tree shaking. Can we make sure we only pull in what we need here? Namely, the UUID parse and stringify functions?

https://www.npmjs.com/package/uuid#uuidparsestr

https://www.npmjs.com/package/uuid#uuidstringifyarr-offset

Reply:
Updated here 1a762f4